internal/drivers/manager.go

```diff
@@ -2,17 +2,19 @@ package drivers
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"runtime/debug"
 	"sort"
 	"sync"
 	"time"
 
+	"github.com/cenkalti/backoff/v4"
```

Comment:
this should be below drone imports in a separate section. See how other files do it in the repo

Reply:
because it's 3rd party

Reply:
ignore you've already done it